src/uct/tcp/tcp_ep.c

```diff
@@ -610,6 +642,7 @@ static inline ssize_t uct_tcp_ep_sendv(uct_tcp_ep_t *ep)
             return 0;
         }
 
+        status = uct_tcp_ep_handle_send_err(ep, status);
```

Comment:
we we can both return error here, and also invoke completion?

Reply:
done